src/ontology/mondo-ingest.Makefile

```diff
@@ -371,6 +371,12 @@ $(TMPDIR)/mondo.sssom.tsv: $(TMPDIR)/mondo_repo_built
 $(TMPDIR)/mondo.owl: $(TMPDIR)/mondo_repo_built
 	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
 
+$(TMPDIR)/mondo-edit.owl: $(TMPDIR)/mondo_repo_built
+	$(ROBOT) convert -i $(TMPDIR)/mondo/src/ontology/mondo-edit.obo -o $@
+
+$(TMPDIR)/mondo-edit-mondo-owl-merged.owl: $(TMPDIR)/mondo-edit.owl $(TMPDIR)/mondo.owl
+	$(ROBOT) merge -i $(TMPDIR)/mondo-edit.owl -i $(TMPDIR)/mondo.owl convert -o $@
```

Comment:
Ah I see, you merged them, not pass both as inputs. That makes sense.

Reply:
@matentzn Coming back to this. Trying to remember fundamentally why we ought to merge these.

I know that `mondo.owl` ends up removing redundant subclass axioms, and we want to access `mondo-edit` so that we have access to those.

But what does `mondo.owl` add that we need to use for the subclass sync which `mondo-edit` does not already have?

Reply:
> But what does mondo.owl add that we need to use for the subclass sync which mondo-edit does not already have?

Inferred subclass relations based on equivalence class axioms for starters!

Reply:
right, dumb qustion